pytapo/__init__.py

```diff
@@ -305,13 +305,6 @@ def setLEDEnabled(self, enabled):
             }
         )
 
-    def setDayNightMode(self, inf_type):
-        if inf_type not in ["off", "on", "auto"]:
-            raise Exception("Invalid inf_type, can be off, on or auto")
-        return self.performRequest(
-            {"method": "set", "image": {"common": {"inf_type": inf_type}}}
-        )
-
```

Comment:
Moved this further down the file to be near the other "set image common" method.